services/tokei/tokei-github-lines.service.js

```diff
@@ -0,0 +1,65 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { metric } = require('../text-formatters')
+const { nonNegativeInteger } = require('../validators')
+const { BaseJsonService } = require('..')
+
+const schema = Joi.object({
+  lines: nonNegativeInteger,
+}).required()
+
+const documentation = `
+<p>
+  The <code>provider</code> is the domain name of git host.
+  If no TLD is provided, <code>.com</code> will be added.
+  For example, setting <code>gitlab</code> or <code>bitbucket.org</code> as the provider also works.
+</p>
+`
+
+module.exports = class LinesOfCode extends BaseJsonService {
```

Comment:
I think it's reasonably likely that we'd get enhancement requests for this that tokei also supports (like `files` for example), so we should probably just use something like `Tokei` for the class name

```suggestion
module.exports = class Tokei extends BaseJsonService {
```

Reply:
This also applies for me to the file name (it'd be confusing to have `lines` in the file name as well if/when other metrics are supported)